### PR TITLE
Add a story and examples describing manual tests

### PR DESCRIPTION
**Pull request by psss**
There are very different naming conventions regarding description of the individual test steps, here are some for the initial inspiration:

* nitrate — setup, actions, expected results, break down
* polarion — setup, test steps (step, expected result), tier down
* xray — pre-conditions, steps (step, data, expected result)
* codebeamer — pre-action, test steps (action, expected result), post-action

### CODE REVIEW

**Comment by psss** [comment]
@Athwale, @psklenar, here's the first draft of how the manul test cases could look like. Please review and comment.

**Comment by Athwale** [comment]
We use Nitrate. For storing two types of manual test cases. 
1st type:
Is manual only. There are no scripts in git for such test cases. Only description in Nitrate. As an example please check out cases for cups in tcms for example case 11652, but there are many others there.

2nd type:
There are manual steps described in nitrate along with runable scripts in git. These tests usually require special machines or manual input due to impossibility of automating the steps. Again as an example check out chrony cases in nitrate, for example case 590298.

There may be other types that I have not seen yet.

File: examples/manual/share.fmf [hunk 1]
```diff
@@ -0,0 +1,56 @@
+summary: A set of manual tests with common setup and cleanup
+
+description:
+    Demonstrates how fmf inheritance can be used to prevent
+    unnecessary duplication of setup and cleanup details.
+
+manual:
+    setup:
+        Install latest tmt package from the copr repository.
```
Comment (pvalena) [inline examples/manual/share.fmf:9]: Could we treat this as a bash script, if we want to run this in semi-automated way later on?
```suggestion
        # Install latest tmt package from the copr repository.
```

Comment (psss) [reply examples/manual/share.fmf:9]: Interesting idea. Yes, I think for semi-automated workflows something like that could be useful.

Comment (psss) [reply examples/manual/share.fmf:9]: Updated the example with your suggestion.

**Comment by psss** [comment]
@Athwale, thanks for providing examples. I've quickly looked through them and it seems to me there should be no problem with the conversion. The manual test execution data fields would be filled with the right content. When executing semi-automated tests you would be able to run scripts which are already in the git repository. So everything is at one place. Do you see anything which could be a problem?

In the future, as @pvalena suggested, we could also support semi-automated tests in a better way, for example proposing `setup` step as a shell script for execution, then doing some manual verification and finally performing the cleanup based on the script in the `cleanup` attribute. Does it make sense to you?

**Comment by Athwale** [comment]
@psss Seems OK to me.

**Comment by psklenar** [comment]
@psss LGTM

**Comment by psss** [comment]
Thanks for the review.

**Pull request merged**